pkgs/development/python-modules/pypandoc/default.nix

```diff
@@ -3,21 +3,21 @@
 
 buildPythonPackage rec {
   pname = "pypandoc";
-  version = "unstable-2018-06-18";
+  version = "1.5";
 
   src = fetchFromGitHub {
     owner = "bebraw";
     repo = pname;
-    rev = "87912f0f17e0a71c1160008df708c876d32e5819";
-    sha256 = "0l6knkxxhmni4lx8hyvbb71svnhza08ivyklqlk5fw637gznc0hx";
+    rev = version;
+    sha256 = "1lpslfns6zxx7b0xr13bzg921lwrj5am8za0b2dviywk6iiib0ld";
   };
 
   postPatch = ''
     # set pandoc path statically
     sed -i '/^__pandoc_path = None$/c__pandoc_path = "${pandoc}/bin/pandoc"' pypandoc/__init__.py
 
     # Fix tests: requires network access
-    substituteInPlace tests.py --replace "pypandoc.convert(url, 'html')" "'GPL2 license'"
+    substituteInPlace tests.py --replace "pypandoc.convert_file(url, 'html')" "'GPL2 license'"
```

Comment:
This changes [the test case](https://github.com/bebraw/pypandoc/blob/1.5/tests.py#L182-L185)
```python
    def test_basic_conversion_from_http_url(self):
        url = 'https://raw.githubusercontent.com/bebraw/pypandoc/master/README.md'
        received = pypandoc.convert_file(url, 'html')
        assert "GPL2 license" in received
```
to
```python
    def test_basic_conversion_from_http_url(self):
        url = 'https://raw.githubusercontent.com/bebraw/pypandoc/master/README.md'
        received = 'GPL2 license'
        assert "GPL2 license" in received
```
... thereby defeating the purpose of that test case. Wouldn't it be more prudent to skip or completely remove that test case?

Reply:
Skipping also kinda does defeat its purpose :) Anyways, I reworked it to skip the test now, hopefully the `sed` call doesn't break in the future.

Reply:
> Skipping also kinda does defeat its purpose :)

Yeah, but when skipped, the test case won't show up as "passed" when it didn't actually test what it's supposed to test. :grin: